airflow/api_connexion/endpoints/task_instance_endpoint.py

```diff
@@ -378,11 +386,15 @@ def get_task_instances(
         )
         .add_columns(SlaMiss)
         .options(joinedload(TI.rendered_task_instance_fields))
-        .offset(offset)
-        .limit(limit)
     )
+
+    order_by_params = _get_order_by_params(order_by or "start_date")
```

Comment:
This default should be mentioned somewhere. I’m not sure this is a good default though? I’d probably expect this to be ordered naturally (the current behaviour in main), or by `(dag_id, dag_run_id, task_id, map_idnex)`.

Reply:
Adding this to the REST API docs. Thanks.

Reply:
Keeping `order_by` to default to `map_index` (same as it has been for Get Mapped Instances) for consistency. User can specify the `order_by` param as required by referring mentioned to the docs.